addons/website_crm_partner_assign/controllers/main.py

```diff
@@ -242,7 +243,16 @@ def _get_partners_values(self, country=None, grade=None, page=0, references_per_
         if not request.env.user.has_group('website.group_website_restricted_editor'):
             base_partner_domain += [('grade_id.website_published', '=', True)]
         if search:
-            base_partner_domain += ['|', ('name', 'ilike', search), ('website_description', 'ilike', search)]
+            base_partner_domain += expression.OR([
```

Comment:
Note you should now use `Domain` (probably with `Domain(name, 'ilike', search) |` syntax)